NDTensors/src/lib/Sectors/test/test_sector.jl

```diff
@@ -0,0 +1,147 @@
+import NDTensors.Sectors: ⊗, ⊕, ×, Fib, data, Ising, Sector, SU, SU2, U1, Z
+using Test
+
+@testset "Test Named Sectors" begin
+  @testset "Construct from × of NamedTuples" begin
+    s = (A=U1(1),) × (B=SU2(2),)
+    @test length(s) == 2
+    @test s[:A] == U1(1)
+    @test s[:B] == SU2(2)
+
+    s = s × (C=Ising("ψ"),)
+    @test length(s) == 3
+    @test s[:C] == Ising("ψ")
+  end
+
+  @testset "Construct from × of Pairs" begin
+    s = ("A" => U1(1)) × ("B" => SU2(2))
```

Comment:
Is the version using `Pair` needed if we have the `NamedTuple` syntax? Just trying to be conservative about the interface for now.

Reply:
No problem to remove this –